site/content/support.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Where to Go for Support"
```

Comment:
```suggestion
title: "Support"
```
The previous title is a bit of a run-on sentence viewed in the lens of a navigable, top-level item in the information architecture.

I can't decide where I would want this to sit, either. None of the other NGINX documentation sets I looked at seemed to have an equivalent document.  My inclination is underneath "How-To Guides" in the existing IA, but the NGINX Ingress Controller equivalent I would put into the Troubleshooting section.